scylla/src/transport/retry_policy.rs

```diff
@@ -686,4 +684,456 @@ mod tests {
             RetryDecision::DontRetry
         );
     }
+
+    const CLS: &[Consistency] = &[
```

Comment:
we can afford longer names - let's expand it to CONSISTENCY_LEVELS or whatever the acronym means